models/truckModel.js

```diff
@@ -2,14 +2,19 @@
 // when exporting the mongoose model you will create using this schema, 
 // name your model 'Truck' (see the example in the slide deck for 'Book')
 // this is to match the name of the collection the data was imported into
+var mongoose = require('mongoose');
+var Schema = mongoose.Schema;
+var db = mongoose.connect('mongodb://localhost/foodTruckApi');
```

Comment:
Database name should be foodTruckAPI (since I'm running student code against my database, the name has to match my database.)